go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/registrobr/rdap-client
+
+go 1.14
+
+require (
+	github.com/aryann/difflib v0.0.0-20121123051704-035af7c09b12
+	github.com/codegangsta/cli v1.19.2-0.20170424213122-ab403a54a148
```

Comment:
Maybe is better set `v1.22.4`?

Reply:
v1.22.5 already and also update it to github.com/urfave/cli